include/swift/AST/LazyResolver.h

```diff
@@ -83,7 +87,7 @@ class alignas(void*) LazyMemberLoader {
 
   /// Populates a vector with all members of \p IDC that have DeclName
   /// matching \p N.
-  virtual TinyPtrVector<ValueDecl *>
+  virtual llvm::TinyPtrVector<ValueDecl *>
```

Comment:
FYI you can include the `swift/Basic/LLVM.h` header to allow these types to be directly referenced in the `swift` namespace